lib/taste_tester/commands.rb

```diff
@@ -63,6 +63,69 @@ def self.status
       end
     end
 
+    def self.run_parallel(mode, hosts, repo = nil)
+      host_threads = []
+      tested_hosts = []
+      connect_failures = 0
+      server = TasteTester::Server.new
+      hosts.each do |hostname|
+        # Poor man thread pool manager: keeping it simple
+        nb_threads_over_max = host_threads.length - TasteTester::Config.parallel_hosts
+        if nb_threads_over_max >= 0
+          host_threads[nb_threads_over_max].join
```

Comment:
shouldn't this `join` be rescued as well?

Reply:
Good point. I guess this never happened in my testing 'cause I have a gazillion of cores :)